README.md

```diff
@@ -39,11 +39,11 @@ for a full build,
 make SUBTARGET=tiny
 ```
 
-for a build including a small subset of supported systems.
+for a build that includes a small subset of supported systems.
 
 See the [Compiling MAME](http://docs.mamedev.org/initialsetup/compilingmame.html) page on our documentation site for more information, including prerequisites for macOS and popular Linux distributions.
 
-For recent versions of macOS you need to install [Xcode](https://developer.apple.com/xcode/) including command-line tools and [SDL 2.0](https://github.com/libsdl-org/SDL/releases/latest).
+For recent versions of macOS, you need to install [Xcode](https://developer.apple.com/xcode/), which includes command-line tools, and [SDL 2.0](https://github.com/libsdl-org/SDL/releases/latest).
```

Comment:
This is inaccurate - you've changed the meaning.  The command-line tools must be included when installing.  It isn't supposed to say that an Xcode installation automatically includes the tools.